rebar.config

```diff
@@ -45,7 +45,7 @@
             {config, "config/test.config"}
         ]}
     ]},
-    {doc, [
+    {docs, [
```

Comment:
Here you are reverting the change in profile name, why?

Reply:
if ex_doc applies `docs` as profile, triggering our DOC define then `rebar3 ex_doc` is enough and we do not need a `doc` alias to inject the profile. I tested and it looks like it is like this. Please remove the unused doc alias.

Reply:
That indeed works - much simpler 8)